src/applications/medical-copays/components/HTMLStatementList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation, Link } from 'react-router-dom';
+import moment from 'moment';
+import { sortStatementsByDate } from '../utils/helpers';
+
+const HTMLStatementList = () => {
+  const { pathname } = useLocation();
+  const selectedId = pathname.replace('/balance-details/', '');
```

Comment:
There must be a better way to get the URL param.

Reply:
Passing in ID instead of pulling from URL since that's all this was doing